security/pkg/util/configutil_test.go

```diff
@@ -31,98 +31,8 @@ import (
 const (
 	configMapName = "test-configmap-name"
 	namespaceName = "test-ns"
-	dataName      = "test-data-name"
 )
 
-func TestInsertDataToConfigMap(t *testing.T) {
```

Comment:
TestInsertDataToConfigMap() is a unit test for InsertDataToConfigMap(); this unit test should be kept.

Reply:
I was going to fix this but this unit test is exactly identical to the one below it, we get no additional coverage from having it

Reply:
I can add it back if you want though

Reply:
It will improve test coverage when each function has a unit test. Though TestInsertDataToConfigMap() and TestInsertDataToConfigMapWithRetry() are mostly the same, they are for two different functions: InsertDataToConfigMap() and InsertDataToConfigMapWithRetry().

Reply:
It does not increase coverage in terms of lines of code executed. For example, If I have two functions:

`b() { a() }` and `a() { do something }`

Calling `b()` will execute the same lines of code as calling both `b()` and `a()`. This is essentially what we have here

Reply:
b() and a() are two different functions; they still should have their own unit tests. If b() and its unit test is deleted one day, a() and its unit test may still exist.

Reply:
I will agree to disagree :slightly_smiling_face: I will add it back